.netlifyredirects

```diff
@@ -0,0 +1,4 @@
+/internal   /api
+/tomster/*  /mascots/:splat
+/dasboard   /statusboard
+/guides/*   https://guides.emberjs.com/:splat
```

Comment:
```suggestion
/guides/*   https://guides.emberjs.com/release/:splat
```

Looking at one of the examples of a "bad link" that this is supposed to fix https://emberjs.com/guides/models/ it seems like we need to include `release` in the redirect 🤔 

without this improvement https://deploy-preview-133--ember-beta-website.netlify.com/guides/models/ redirects to https://guides.emberjs.com/models/ which is a 404